### ISSUE TITLE
[traderepublic]

### ISSUE THREAD

**Issue by codeofsumit** [codeofsumit/tresor-import#294]
## Description
> test

## Document
<details><summary>pb16062276175996321565930392605.pdf</summary><p>

```json
[
  [
    "TRADE REPUBLIC BANK GMBH",
    "KASTANIENALLEE 32",
    "10435 BERLIN",
    "Dominique Dennis Sonntag",
    "Rosenheimer Straße 90",
    "81669 München",
    "SEITE",
    "1",
    "von",
    "1",
    "DATUM",
    "24.11.2020",
    "ORDER",
    "89cd-43b3",
    "DEPOT",
    "0110980801",
    "Trade Republic Bank GmbH",
    "Kastanienallee 32",
    "10435 Berlin",
    "www.traderepublic.com",
    "sxzfhwj@jznksesofyses.iry",
    "Sitz der Gesellschaft: Düsseldorf",
    "AG Düsseldorf HRB 85864",
    "USt-ID DE307510626",
    "Geschäftsführer",
    "Andreas Willius",
    "Karsten Müller",
    "/ 51b3-d409",
    "AUFTRAGSBESTÄTIGUNG STOP-ORDER VERKAUF",
    "ÜBERSICHT",
    "Bestätigung der Erstellung der Stop-Order Verkauf mit der Ordernummer 89cd-43b3 vom 24.11.2020",
    "um 15:20 Uhr.",
    "WERTPAPIER",
    "NOMINALE",
    "STOP",
    "GÜLTIG BIS",
    "HANDELSPLATZ",
    "Alibaba",
    "ISIN: US01609W1027",
    "5 Stk.",
    "223 EUR",
    "19.11.2021",
    "Lang und Schwarz Exchange"
  ]
]

```

</p></details>

## Community profile

https://app.tresor.one/p/5fc0fbd037686e003820ac2a

---
Issue submitted with https://import.tresor.one

**Comment by SirGibihm** [comment]
Wrong document to be imported, this is only the confirmation that an order has been placed, not the document that contains an executed order. This I think we do not want to implement this. If you agree @dredav please close this issue :)

**Comment by dredav** [comment]
@SirGibihm Yes and no, see my comment here from #280:
>>Yes we dont wan't to parse cost informations - but I think we should set a specific status code for cost informations. The UI can display some information like:

>Kosteninformationen können nicht als Aktivität importiert werden, da die Order noch nicht ausgeführt wurde.

Additional: When Sumit don't want to show the result of the status code, we can show this information at the import-tool too. Thats the "last" hurdle before the user submit his document to us.

**Comment by SirGibihm** [comment]
> 
> 
> @SirGibihm Yes and no, see my comment here from #280:
> 
> > > Yes we dont wan't to parse cost informations - but I think we should set a specific status code for cost informations. The UI can display some information like:
> 
> > Kosteninformationen können nicht als Aktivität importiert werden, da die Order noch nicht ausgeführt wurde.
> 
> Additional: When Sumit don't want to show the result of the status code, we can show this information at the import-tool too. Thats the "last" hurdle before the user submit his document to us.

I am not familiar with the UI Side of the import tool at all so I can't really make a statement on this

**Comment by dredav** [comment]
Document will ignored with status code `7`. Solved in #524
